rbac/providers/azure/delta_inbound_sync.py

```diff
@@ -154,7 +155,7 @@ def insert_change_to_db(data, record_timestamp):
         "timestamp": record_timestamp,
         "provider_id": TENANT_ID,
     }
-    response = r.table("queue_inbound").insert(inbound_entry).run()
+    r.table("queue_inbound").insert(inbound_entry).run()
```

Comment:
I'm guessing you'll want to put this in a try catch, and also check the result of run.
I believe it will throw an exception if rethink is not available, and also it will *not* throw an exception if there is an insertion error.